draft-ietf-quic-recovery.md

```diff
@@ -1157,6 +1095,9 @@ window.
        congestion_window *= kLossReductionFactor
        congestion_window = max(congestion_window, kMinimumWindow)
        ssthresh = congestion_window
+       // Collapse congestion window if persistent congestion
+       if (pto_count > 2):
```

Comment:
2 is a magic number.  Previously we said you SHOULD have 2 TLPs, but you could have fewer.  I think it makes sense to define this as a constant and explain its reason for existing there?

Reply:
I commented on the thread above this. Yes we should settle on one or two TLPs. I personally prefer one but unfortunately I don't have data to say if two is better since Windows TCP has been doing just one TLP.

Reply:
@siyengar : In this case, by the time 5 is acked, 3 and 4 should be marked as lost by the time-threshold loss detection mechanism, since they're going to be > 1.125 x SRTT in the past.